.circleci/continue_config.yml

```diff
@@ -0,0 +1,576 @@
+version: 2.1
+
+orbs:
+  maven: circleci/maven@1.2.0
+
+jobs:
+  lms_rails_build:
+    parameters:
+      should-run:
+        type: boolean
+        default: false
+    working_directory: ~/Empirical-Core
+    parallelism: 6
+    docker:
+      - image: cimg/ruby:3.1.4
+        environment:
+          PG_DB: quill_test_db
+          PG_USER: ubuntu
+          RAILS_ENV: test
+          RACK_ENV: test
+      - image: pgvector/pgvector:pg15
+        environment:
+          POSTGRES_DB: quill_test_db
+          POSTGRES_USER: ubuntu
+          POSTGRES_PASSWORD: password
+      - image: pgvector/pgvector:pg15
+        name: test_replica
+        environment:
+          POSTGRES_DB: quill_test_db
+          POSTGRES_USER: ubuntu
+          POSTGRES_PASSWORD: password
+    steps:
+      - run:
```

Comment:
Note, jobs must have one step. So in case the condition is false, include a blank step.

Reply:
This feels like something that should be captured in the configuration itself.  Maybe replace each of the `echo 'hello'` sections with:

```
- run:
  name: Required first step in case proceeding condition is false
  command: echo 'about to check run conditions'
```